src/System.Management.Automation/engine/MshObject.cs

```diff
@@ -2437,19 +2394,7 @@ internal PSMemberInfoInternalCollection<PSPropertyInfo> ClrMembers
         /// </summary>
         internal PSPropertyInfo GetFirstPropertyOrDefault(MemberNamePredicate predicate)
         {
-            if (_instanceMembers != null)
```

Comment:
`_instanceMembers` could be null because it has been revived yet. It's safer to use the property `InstanceMebmers`.
This is not needed anymore as the instance members are handled in `FirstOrDefault`.